java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -664,4 +667,51 @@ public void flushall() {
                         .toLowerCase()
                         .contains("can't write against a read only replica"));
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void functionLoad(boolean withRoute) {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0"), "This feature added in redis 7");
+        String libName = "mylib1C" + withRoute;
+        String code =
+                "#!lua name="
+                        + libName
+                        + " \n redis.register_function('myfunc1c"
+                        + withRoute
+                        + "', function(keys, args) return args[1] end)";
+        Route route = new SlotKeyRoute("1", PRIMARY);
+
+        var promise =
+                withRoute ? clusterClient.functionLoad(code, route) : clusterClient.functionLoad(code);
+        assertEquals(libName, promise.get());
+        // TODO test function with FCALL when fixed in redis-rs and implemented
+        // TODO test with FUNCTION LIST
```

Comment:
can you use custom command for LIST?

Reply:
Next PR #277 comes to help you with that